gem/middleware.py

```diff
@@ -54,6 +55,25 @@ class GemMoloGoogleAnalyticsMiddleware(MoloGoogleAnalyticsMiddleware):
     into the MoloGoogleAnalyticsMiddleware class and override
     submit_to_local_account
     '''
+    def get_article_tags(self, request):
+        """get the tags in an article if the request is for an article page"""
+        tags = []
+        path = request.get_full_path()
+        path_components = [component for component in path.split('/')
+                           if component]
+
+        try:
+            page, args, kwargs = request.site.root_page.specific.route(
+                request,
+                path_components)
+            if issubclass(type(page), ArticlePage):
+                tags = page.tags_list()
+                print('|'.join(tags))
+            return '|'.join(tags)
+
+        except Http404:
+            raise []
```

Comment:
What are you raising here? did you intend maybe to `return` instead of `raise`?

Reply:
@miltontony yeap, it's suppose to be a return